db.js

```diff
@@ -217,8 +218,16 @@ exports.init = function (sbot, config) {
       stateFeedsReady,
       (ready) => ready === true,
       () => {
+        if (msg.recps) {
+          msg = ssbKeys.box(
+            msg,
+            msg.recps.map((x) => x.substr(1))
```

Comment:
This only supports classic encryption. A good start but we need an interface like add Boxer and addUnboxer right?

Reply:
Yes, but I would rather do that in https://github.com/ssb-ngi-pointer/ssb-db2/pull/90. Would like https://github.com/ssb-ngi-pointer/ssb-fixtures/pull/2 to be in a bit better shape before really looking at that again though.

Reply:
Hmm, I was looking at that `x.substr(1)` and got confused. Do we need to remove the sigil `@`?

According to ssb-keys implementation, yes: 

https://github.com/ssb-js/ssb-keys/blob/98569aa430e344b3ed144fcebf5d2755f7355bf6/index.js#L139

But according to how Patchwork does it right now, it checks if it's a `Ref.isFeed()`:

https://github.com/ssbc/patchwork/blob/ae29bd7f91e16deac4bd88b099175fa85a419815/lib/depject/sbot.js#L172-L174

How does Patchwork even work?

Reply:
@staltz https://github.com/ssb-js/ssb-keys/pull/92 ;-)

Reply:
Given that it also works that way I'm open to maybe doing this without the substr. What do you think?

Reply:
Yeah this substr here seems not necessary if ssb-keys does it in utils.toBuffer